Dockerfile

```diff
@@ -13,6 +13,8 @@ COPY resources/setup.yml /etc/bahmni-installer/setup.yml
 COPY inventories/${INVENTORY_FILE} /etc/bahmni-installer/local
 COPY resources/install_bahmni.sh /tmp
 COPY resources/start_bahmni.sh /tmp
+COPY resources/odoo.conf /tmp
+COPY resources/odoo /tmp
 
```

Comment:
@chinzou , do you think using the **/tmp/** folder here could be a problem?
I mean in case of restart, we may just loose those utils files isn't it?

Reply:
the files in **/tmp** are not getting deleted in case of restart, do you think that we can have a better location?